core/src/main/java/hudson/slaves/ChannelPinger.java

```diff
@@ -42,7 +42,7 @@
 import java.util.logging.Logger;
 
 /**
- * Establish a periodic ping to keep connections between {@link Slave slaves}
+ * Establish a periodic ping to keep connections between {@link Agent agent}
```

Comment:
This is a link to another class, see http://javadoc.jenkins.io/hudson/slaves/ChannelPinger.html

Replacing this will break the link.

Reply:
Fixed